SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -66,8 +193,36 @@ bool FRPCContainer::ObjectHasRPCsQueuedOfType(const Worker_EntityId& EntityId, E
 
 	return false;
 }
+ 
+void FRPCContainer::BindProcessingFunction(const FProcessRPCDelegate& Function)
+{
+	ProcessingFunction = Function;
+}
 
-bool FRPCContainer::ApplyFunction(const FProcessRPCDelegate& FunctionToApply, const FPendingRPCParams& Params)
+bool FRPCContainer::ApplyFunction(FPendingRPCParams& Params)
 {
-	return FunctionToApply.Execute(Params);
+	FRPCErrorInfo ErrorInfo;
+
+	if (ProcessingFunction.IsBound())
+	{
+		ErrorInfo = ProcessingFunction.Execute(Params);
+	}
+	else
+	{
+		ErrorInfo = FRPCErrorInfo{ nullptr, nullptr, true, ERPCQueueType::Unknown, ERPCResult::NoProcessingFunctionBound };
+	}
+
+	if (ErrorInfo.Success())
+	{
+		return true;
+	}
+	else
+	{
+#if !UE_BUILD_SHIPPING
+		bool bDropRPC = false;
+		LogRPCError(ErrorInfo, Params, bDropRPC);
```

Comment:
Should we remove the `bDropRPC` argument for now since it's always passed as false and add it back if we start using it?

Reply:
Removed.